packages/xo-web/src/xo-app/new-vm/index.js

```diff
@@ -1208,13 +1222,39 @@ export default class NewVm extends BaseComponent {
               </span>
             </LineItem>
             <br />
-            <DebounceTextarea
-              className='form-control'
-              disabled={installMethod !== 'customConfig'}
-              onChange={this._linkState('customConfig')}
-              rows={7}
-              value={defined(customConfig, DEFAULT_CLOUD_CONFIG_TEMPLATE)}
-            />
+            <LineItem>
+              <label className='text-muted' htmlFor='customConfig'>
```

Comment:
Why `htmlFor`?

Reply:
https://reactjs.org/docs/dom-elements.html#htmlfor

Reply:
Not necessary:

> You can nest the <input> directly inside the <label>, in which case the for/id attributes are not needed because the association is implicit.
>
> -- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label

Reply:
Also static identifiers should be avoided in React because they need to be unique while React components can be multiple.